ajax/libs/vivagraphjs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "vivagraphjs",
+  "version": "0.8.2",
+  "description": "Graph Drawing Library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anvaka/VivaGraphJS.git"
+  },
+  "keywords": [
+    "graph"
+  ],
+  "author": "Andrei Kashcha",
+  "license": "BSD-3-Clause",
+  "npmName": "vivagraphjs",
+  "npmFileMap": [
+    {
+      "basePath": "",
```

Comment:
We need all files (which corresponds to the rule `**/*`) under `dist` directory, so let's use `dist` as the `basePath` :smiley: